src/GafferSceneUI/SceneGadget.cpp

```diff
@@ -378,6 +378,10 @@ void SceneGadget::setRenderer( IECore::InternedString name )
 		newRenderer
 	);
 
+	// OutputBuffer will set up outputs directly in the renderer - the controller won't know about these
+	// outputs, so we must manually enable the manifest so that we get ids.
+	newController->setManifestRequired( true );
```

Comment:
We only want to do this when we actually have an OutputBuffer. Otherwise we're paying for stuff we don't use in the OpenGL render.